api/bots/giphy/giphy.py

```diff
@@ -12,9 +12,10 @@
 
 GIPHY_TRANSLATE_API = 'http://api.giphy.com/v1/gifs/translate'
 
-if not os.path.exists(os.environ['HOME'] + '/.giphy_config'):
-    print('Giphy bot config file not found, please set up it in ~/.giphy_config'
-          '\n\nUsing format:\n\n[giphy-config]\nkey=<giphy API key here>\n\n')
+if not os.path.exists(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'giphy.conf')):
+    print('Giphy bot config file not found, please set it up in this bot\'s folder '
```

Comment:
Maybe we want this printed in separate `print()` statements? I think it could noticeably increase the readability.